src/hooks/useFilterPossiblyMaliciousPositions.ts

```diff
@@ -0,0 +1,56 @@
+import { Token } from '@uniswap/sdk-core'
+import { useMemo } from 'react'
+import { PositionDetails } from 'types/position'
+import { hasURL } from 'utils/urlChecks'
+
+import { useDefaultActiveTokens } from './Tokens'
+import { useTokenContractsConstant } from './useTokenContractsConstant'
+
+function getUniqueAddressesFromPositions(positions: PositionDetails[]): string[] {
+  return Array.from(
+    new Set(positions.reduce<string[]>((acc, position) => acc.concat(position.token0, position.token1), []))
+  )
+}
+
+export function useFilterPossiblyMaliciousPositions(positions: PositionDetails[]): PositionDetails[] {
+  const activeTokensList = useDefaultActiveTokens()
+
+  const nonListPositionTokenAddresses = useMemo(
+    () => getUniqueAddressesFromPositions(positions).filter((address) => !activeTokensList[address]),
+    [positions, activeTokensList]
+  )
+
+  const symbolCallStates = useTokenContractsConstant(nonListPositionTokenAddresses, 'symbol')
```

Comment:
nit: what does `call states` mean in this context? To me, this is just `nonListTokenSymbols`

Reply:
this is an array of async action states from chain calls

```
interface CallState {
    readonly valid: boolean;
    readonly result: CallStateResult | undefined;
    readonly loading: boolean;
    readonly syncing: boolean;
    readonly error: boolean;
}
```